joboard_home/templates/joboard_home/home_page.html

```diff
@@ -48,85 +49,95 @@
           </svg> Sign In </button>
 
-          <button onclick="location.href='{% url 'accounts:signup' %}'" class="btn btn-sm btn-outline-secondary" 
-          type="button"><svg width="1em" height="1em" viewBox="0 0 16 16" 
+          <button onclick="location.href='{% url 'accounts:signup' %}'" class="btn btn-sm btn-outline-secondary"
+          type="button"><svg width="1em" height="1em" viewBox="0 0 16 16"
           class="bi bi-person-plus-fill" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
           <path fill-rule="evenodd" d="M1 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H1zm5-6a3 3 0 1 0 0-6 3 3 0 0 0
-           0 6zm7.5-3a.5.5 0 0 1 .5.5V7h1.5a.5.5 0 0 1 0 1H14v1.5a.5.5 0 0 1-1 
+           0 6zm7.5-3a.5.5 0 0 1 .5.5V7h1.5a.5.5 0 0 1 0 1H14v1.5a.5.5 0 0 1-1
            0V8h-1.5a.5.5 0 0 1 0-1H13V5.5a.5.5 0 0 1 .5-.5z"/>
           </svg>Sign Up</button>
         {% endif %}
       </div>
 
-
-
+      <img class="logo" src="{% static 'images/sitelogovector.svg' %}">
 
       <div class="header">
-         <form>
-         <img class="logo" src="{% static 'images/sitelogovector.svg' %}">
-         <h1><strong>   JoBoard </strong></h1>
-         <h2><strong>One stop to your next job!</strong></h2>
-     <div class= "form-box">
-          <input type="text" class="search-field job"
-          placeholder = "what is your profession?">
-
-          <input type="text" class="search-field location"
-          placeholder = " where?">
-
-          <button class="search-btn" type="button"><i class="fas fa-search"></i>
-          <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-search"
-           fill="currentColor" xmlns="http://www.w3.org/2000/svg">
-           <path fill-rule="evenodd" 
-           d="M10.442 10.442a1 1 0 0 1 1.415 0l3.85 3.85a1 1 0 0 1-1.414 1.415l-3.85-3.85a1 1 0 0 1 0-1.415z"/>
-           <path fill-rule="evenodd"
-            d="M6.5 12a5.5 5.5 0 1 0 0-11 5.5 5.5 0 0 0 0 11zM13 6.5a6.5 6.5 0 1 1-13 0 6.5 6.5 0 0 1 13 0z"/>
-           </svg>find it! </button>
-            
-       </div>
+         <form action="/jobs/" method="POST" >
+                 {% csrf_token %}
+                <h1><strong> JoBoard </strong></h1>
+                <h2><strong>One stop to your next job!</strong></h2>
+            <div class= "form-box">
+                <input name="JobType" type="text" class="search-field job"
+                placeholder = "what is your profession?" title="what is your profession?">
+
+                <input name="JobCity" type="text" class="search-field location"
+                placeholder = " where?" title="where?">
+
+                <select name="JobScope" class="search-field scope" title="choose the job scope">
+                <option value disabled selected>choose the job scope</option>
+                <option value="FULL">FULL</option>
+                <option value="PART">PART</option>
+                <option value="INTER">INTER</option>
```

Comment:
what is INTER? (you mean intern?)

Reply:
yes :(
should  I need to change the mistake in the models as well as part of this PR?
Is it possible to merge this PR and I will fix the mistake over all the repo in a new PR?

Reply:
Theoretically it's possible, but best practice is not to merge wrong things and then fix them in a future PR.
You can either create another PR in which you fix that everywhere, and then rebase this PR on top of it, or actually just remove it since intern and student (the next option on line 125) are the same I think

Reply:
Done :)